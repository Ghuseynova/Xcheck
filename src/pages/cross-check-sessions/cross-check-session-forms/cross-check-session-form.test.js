import React from 'react';
import { shallow } from 'enzyme';
import CrossCheckSessionForm from './cross-check-session-form';

describe('render ', () => {
  test('CrossCheck session form', () => {
    const tree = shallow(<CrossCheckSessionForm/>);
	  expect(tree).toMatchSnapshot();
	});
});