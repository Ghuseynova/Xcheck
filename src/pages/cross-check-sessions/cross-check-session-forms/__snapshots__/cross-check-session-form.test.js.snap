// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render  CrossCheck session form 1`] = `ShallowWrapper {}`;
